From 8235d9ec36720e72cacec2721d67a8682149c677 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heiko.becker@kde.org>
Date: Fri, 13 Jun 2025 15:55:03 +0200
Subject: [PATCH] Drop no longer used icu dependency

It's no longer necessary since commit
6fdca0deaebb9d2adc22c737359e774b618fd358.
---
 CMakeLists.txt                  | 6 ------
 kcms/dateandtime/CMakeLists.txt | 2 --
 2 files changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a05e390797..ef8a77f44f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -233,12 +233,6 @@ if (BUILD_KCM_TOUCHPAD_X11)
 endif()
 
 
-find_package(ICU COMPONENTS i18n uc)
-set_package_properties(ICU PROPERTIES
-    PURPOSE "Retrieving timezone info"
-    TYPE REQUIRED
-)
-
 set_package_properties(XCB PROPERTIES TYPE REQUIRED)
 add_feature_info("XCB-XKB" XCB_XKB_FOUND "Required for building kcm/keyboard")
 add_feature_info("libxft" X11_Xft_FOUND "X FreeType interface library required for font installation")
diff --git a/kcms/dateandtime/CMakeLists.txt b/kcms/dateandtime/CMakeLists.txt
index 347ab280a9..72b866af3c 100644
--- a/kcms/dateandtime/CMakeLists.txt
+++ b/kcms/dateandtime/CMakeLists.txt
@@ -26,8 +26,6 @@ target_link_libraries(kcm_clock
     KF6::KIOCore
     KF6::KIOWidgets
     KF6::Svg
-    ICU::i18n
-    ICU::uc
 )
 install(TARGETS kcm_clock  DESTINATION ${KDE_INSTALL_PLUGINDIR}/plasma/kcms/systemsettings_qwidgets )
 
-- 
GitLab

